parsers/lib/__init__.py

```diff
@@ -0,0 +1 @@
+from .exceptions import ParserException
```

Comment:
I'd personally prefer importing the class in each module that needs it instead of putting it in `__init__.py`.
Any thoughts?